src/dashboard/trade/placeOrder/PlaceOrderContent.tsx

```diff
@@ -0,0 +1,178 @@
+import {
+  createStyles,
+  Grid,
+  makeStyles,
+  Theme,
+  ThemeProvider,
+} from "@material-ui/core";
+import React, { ReactElement, SetStateAction, Dispatch } from "react";
+import { inject, observer } from "mobx-react";
+import ErrorMessage from "../../../common/components/data-display/ErrorMessage";
+import { getErrorMsg } from "../../../common/utils/errorUtil";
+import Loader from "../../../common/components/data-display/loader/Loader";
+import { TradeStore, TRADE_STORE } from "../../../stores/tradeStore";
+import { tradeTheme } from "../../../themes";
+import { isMarketOrder } from "../tradeUtil";
+import AmountDetails from "./AmountDetails";
+import AmountInput from "./AmountInput";
+import OrderSideButtons from "./OrderSideButtons";
+import PlaceOrderButton from "./PlaceOrderButton";
+import PriceInput from "./PriceInput";
+import TotalInput from "./TotalInput";
+import OrderTypeButtons from "./OrderTypeButtons";
+import SectionTitle from "../../../common/components/data-display/text/SectionTitle";
+
+const useStyles = makeStyles((theme: Theme) => {
+  return createStyles({
+    wrapper: {
+      maxWidth: "550px",
```

Comment:
Why is there max width? Imo it does not look good like this. The open orders section does certainly not fit into 550px and would not make sense to keep those sections with different widths. So let's keep them dynamic.
Also, the place order box "jumps" (changes its height) when i switch between buy and sell. The reason is that I have different max limits and with 7 decimal numbers, the `Max: 0.0292936 ETH` does not fit into one row, causing the box to increase in height. At least 8 decimals have to fit into one row.